tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -87,30 +87,30 @@ jobs:
 
 - template: templates/win-ci.yml
   parameters:
-    ort_build_pool_name: 'onnxruntime-gpu-winbuild'
+    ort_build_pool_name: 'onnxruntime-gpu-tensorrt8-winbuild-t4'
```

Comment:
Can we write a comment about why this pool is being used (when using 11.4 some tests are failing on m60 used by the old pool)?

Reply:
i'll add one later. we don't want to reset the CI tests before they finish.